plugins/do_sudo/do_sudo.plugin.zsh

```diff
@@ -38,13 +38,14 @@ function _do_sudo() {
     if [[ ${#cmd_alias_arr[@]} -gt 0 ]]; then
         _do_sudo "${args[@]}" $==*
     else
+        unset __do_sudo_expanded
         if ((__do_sudo_glob)); then
+            unset __do_sudo_glob
             PATH="/sbin:/usr/sbin:/usr/local/sbin:$PATH" command sudo "${args[@]}" $~==*
         else
+            unset __do_sudo_glob
             PATH="/sbin:/usr/sbin:/usr/local/sbin:$PATH" command sudo "${args[@]}" $==*
         fi
-        unset __do_sudo_glob
```

Comment:
Do the same as this was doing...

Reply:
Any suggested solution? Before this change the function always returned 0, which was the exit code of `unset`. A workaround firstly popped up in my mine was to save the exit code of `command sudo ...` in a variable and return it in the end but that's 1 more line than this one.

Reply:
ohhh i see now what you mean.
I didn't catch the fact you were running  a command on those lines after the ones i mentioned...
Those two lines look almost identical, thats why i didn't catch the 1 char diff :smile_cat:

Reply:
No prob. Just force pushed a new commit using a local variable to handle it :)